UnattendResources/Logon.ps1

```diff
@@ -214,6 +214,48 @@ function License-Windows {
     }
 }
 
+function Get-AdministratorAccount {
+    <#
+    .SYNOPSIS
+    Helper function to return the local Administrator account name.
+    This works with internationalized versions of Windows.
+    #>
+    PROCESS {
+        $version = $PSVersionTable.PSVersion.Major
+        if ($version -lt 4) {
+            # Get-CimInstance is not supported on powershell versions earlier then 4
+            New-Alias -Name Get-ManagementObject -Value Get-WmiObject
+        } else {
+            New-Alias -Name Get-ManagementObject -Value Get-CimInstance
+        }
+        $SID = "S-1-5-21-%-500"
+        $modifier = " LIKE "
+        $query = ("SID{0}'{1}'" -f @($modifier, $SID))
+        $s = Get-ManagementObject -Class Win32_UserAccount -Filter $query
+        if (!$s) {
+            throw "SID not found: $SID"
+        }
+        return $s.Name
+    }
+}
+
+function Enable-Administrator {
+    [string]$username = Get-AdministratorAccount
+    $setupCompletePath = "$env:windir\Setup\Scripts\SetupComplete.cmd"
+    $activate = "powershell -c net user {0} /active:yes" -f $username
+    $expiration = 'wmic path Win32_UserAccount WHERE Name="{0}" set PasswordExpires=true' -f $username
+    $logonReset = "net.exe user {0} /logonpasswordchg:yes" -f $username
+    Add-Content -Encoding Ascii -Value $activate -Path $setupCompletePath
+    Add-Content -Encoding Ascii -Value $expiration -Path $setupCompletePath
+    Add-Content -Encoding Ascii -Value $logonReset -Path $setupCompletePath
+    $command = "net.exe user $username """

```

Comment:
you can remove this variable and use the command string directly on the next line.